ja/api/configuration-render.md

```diff
@@ -155,6 +155,8 @@ pushAssets: (req, res, publicPath, preloadFiles) => preloadFiles
 
 > これは Content-Security-Policy で適用された外部リソースを読み込む設定をするために使用します。
 
+NOTE: `script-src` ポリシーに `'unsafe-inline'` が含まれている場合、CSP のハッシュは追加されません。これは、ハッシュが存在する場合、ブラウザが `'unsafe-inline'` を無視するためです。CSPv1 の後方互換性のために `'unsafe-inline'` とハッシュの両方の定義が必要な場合は、オプションの `unsafeInlineCompatiblity` を `true` に設定します。
```

Comment:
```suggestion
`script-src` ポリシーに `'unsafe-inline'` が含まれている場合、CSP のハッシュは追加されないことに注意してください。これは、ハッシュが存在する場合、ブラウザが `'unsafe-inline'` を無視するためです。CSPv1 の後方互換性のために `'unsafe-inline'` とハッシュの両方の定義が必要な場合は、オプションの `unsafeInlineCompatiblity` を `true` に設定します。
```
I think this `Note` is verb. how about this translation?

Reply:
@aytdm 
Exactly. Please review again because the indication is reflected.
https://github.com/nuxt/docs/pull/1573/commits/7f28b204c4bb6fef049a3825ea91ae455cd742e6